spec/models/book_spec.rb

```diff
@@ -0,0 +1,115 @@
+require "rails_helper"
+
+describe Book do
+  before do
+    Book.delete_all
+  end
```

Comment:
Sometime later, we can use [DatabaseCleaner](https://github.com/DatabaseCleaner/database_cleaner) to achieve this.

For now, I'm fine with doing what you've done here.